apps/testclient/management/commands/create_test_users_and_applications_batch.py

```diff
@@ -290,6 +297,18 @@ def create_test_access_refresh_archived_tokens(
         date_archived = ot.created + timedelta(days=10)
         ot.archived_at = date_archived.replace(tzinfo=pytz.utc)
         ot.save()
+        print("<<< " + user.username + " archived token " + str(i) + " generated")
+
+    past_date = timezone.now() - timedelta(days=2)
+    for i in range(archived_grant_count):
+        adag = ArchivedDataAccessGrant.objects.create(beneficiary=user,
```

Comment:
@sharonfruit Why does the data access grant get archived after expiration? Is it just to have the option to renew it if something changes or just for audit purposes?

Reply:
@meliGuzman I believe it is mainly for debugging and audit purposes. It will be especially useful when the data access type feature is fully rolled out because the expiration values can help determine the data access type.